src/utils/reply-middleware.ts

```diff
@@ -0,0 +1,57 @@
+import { FastifyReply } from "fastify"
+import * as WebSocket from "ws"
+
+class ReplyMiddleware {
```

Comment:
little bit confused by a name 'middleware', maybe smth like RpcReply would be better?

Reply:
Agree